system-tests/test/webpack_dev_server_spec.ts

```diff
@@ -1,14 +1,48 @@
 import systemTests from '../lib/system-tests'
+import Fixtures from '../lib/fixtures'
+import path from 'path'
+import globby from 'globby'
+import { escapeRegExp } from 'lodash'
 
 describe('@cypress/webpack-dev-server', function () {
   systemTests.setup()
 
+  const wdsPath = Fixtures.projectPath('webpack-dev-server')
+
   systemTests.it('successfully loads and runs all specs', {
     project: 'webpack-dev-server',
     testingType: 'component',
     spec: 'src/**/*',
     browser: 'chrome',
     expectedExitCode: 0,
+    onRun: async (exec) => {
```

Comment:
Did you consider writing a cy-in-cy test to verify these load? Would have reduced the complexity of the logic below (albeit for a performance cost)

Reply:
@ZachJW34 Was a bit of a toss-up. To do `cypress-in-cypress` I would have had to make more changes to write the 4 special test files this is validating into the c-in-c project which I feel would have obviated much of the advantage. More complex setup balanced against easier validation - decided to leave as much in place as possible and deal with the messy validation to keep the PR simpler and minor perf gain.